target/linux/ramips/mt7620/config-6.6

```diff
@@ -191,12 +199,11 @@ CONFIG_SPI_MASTER=y
 CONFIG_SPI_MEM=y
 # CONFIG_SPI_MT7621 is not set
 CONFIG_SPI_RT2880=y
-CONFIG_SRCU=y
+# CONFIG_SQUASHFS_DECOMP_MULTI_PERCPU is not set
```

Comment:
It seems that some symbols are missing. Please always use `make kernel_menuconfig` to refresh the config file.